phase3/DecisionTreeClassifier.py

```diff
@@ -0,0 +1,205 @@
+import numpy as np
+
+class decisionTreeClassifier:
```

Comment:
Can you rename the file as decisiontreeclassifier.py and the class as DecisionTreeClassifier. That's the format used for imageprocessor and lsh